src/Domain.jl

```diff
@@ -1872,6 +1872,7 @@ end
             flow = inter.F(t)
             @simd for i in domain.indexes[1]:domain.indexes[2]
                 @inbounds @fastmath jac[i,i] -= flow/N
+                @inbounds @fastmath jac[i,:] .+= flow*ns[i]/(N*N)
```

Comment:
This looks right physically and mathematically even if it ruins sparsity XD, but from a simulation standpoint it's kind of terrible, because our most common case involving this is a liquid reactor with an outlet and normally your liquid isn't going to expand to the size of the container it will just effectively be a smaller volume. Not that this needs fixed in this PR, but we should probably think about how we handle an outlet on a liquid reactor because it'd be super easy for someone to setup a liquid reactor simulation with an outlet like this and get a weird result.